repository_service_tuf_worker/signer.py

```diff
@@ -25,20 +25,20 @@ class FileNameSigner(CryptoSigner):
     Provide method to load **unencrypted** PKCS8/PEM private key from file.
 
     File path is constructed by joining base path in environment variable
-    ``RSTUF_ONLINE_KEY_DIR`` with file in ``priv_key_uri``.
+    ``ONLINE_KEY_DIR`` with file in ``priv_key_uri``.
```

Comment:
You said this was done to allow re-using the Dynaconf setting name. Could you give an example where it's possible for `ONLINE_KEY_DIR` name to be reused?

I thought we want to try and keep all `RSTUF` related env variables with the `RSTUF` prefix.

PS: In your changes in `docs/source/guide/Docker_README.md` line 230 we describe with its full name `RSTUF_ONLINE_KEY_DIR`.

Reply:
Then why do we rename it here to `ONLINE_KEY_DIR` only when actually the full name under which you can find it is `RSTUF_ONLINE_KEY_DIR`?

Reply:
I don't got your question.

On the OS/Container we need to use the `RSTUF` :
```shell
root@3f79f6f8a407:/opt/repository-service-tuf-worker# env | grep RSTUF
RSTUF_AWS_ACCESS_KEY_ID=test
RSTUF_AWS_ENDPOINT_URL=http://localstack:4566
RSTUF_BROKER_SERVER=redis://redis/1
RSTUF_SQL_SERVER=postgresql://postgres:secret@db:5432
RSTUF_REDIS_SERVER=redis://redis
RSTUF_AWS_SECRET_ACCESS_KEY=test
RSTUF_STORAGE_BACKEND=AWSS3
RSTUF_AWS_STORAGE_BUCKET=tuf-metadata
RSTUF_AWS_DEFAULT_REGION=us-east-1
```

But, in the Dynaconf it is used only to map these as settings used by Dynaconf

```shell
>>> repository._settings.environ
environ({'RSTUF_AWS_ACCESS_KEY_ID': 'test', 'RSTUF_AWS_ENDPOINT_URL': 'http://localstack:4566', 'HOSTNAME': '3f79f6f8a407', 'PYTHON_VERSION': '3.10.12', 'RSTUF_BROKER_SERVER': 'redis://redis/1', 'RSTUF_SQL_SERVER': 'postgresql://postgres:secret@db:5432', 'PWD': '/opt/repository-service-tuf-worker', 'PYTHON_SETUPTOOLS_VERSION': '65.5.1', 'HOME': '/root', 'LANG': 'C.UTF-8', 'GPG_KEY': 'A035C8C19219BA821ECEA86B64E628F8D684696D', 'RSTUF_REDIS_SERVER': 'redis://redis', 'RSTUF_AWS_SECRET_ACCESS_KEY': 'test', 'TERM': 'xterm', 'SHLVL': '1', 'WORKER_ID': '3f79f6f8a407', 'RSTUF_STORAGE_BACKEND': 'AWSS3', 'PYTHON_PIP_VERSION': '23.0.1', 'PYTHON_GET_PIP_SHA256': '96461deced5c2a487ddc65207ec5a9cffeca0d34e7af7ea1afc470ff0d746207', 'PYTHON_GET_PIP_URL': 'https://github.com/pypa/get-pip/raw/0d8570dc44796f4369b652222cf176b3db6ac70e/public/get-pip.py', 'RSTUF_AWS_STORAGE_BUCKET': 'tuf-metadata', 'PATH': '/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', 'RSTUF_AWS_DEFAULT_REGION': 'us-east-1', 'RSTUF_WORKER_ID': '3f79f6f8a407', '_': '/usr/local/bin/python'})
```

But it is *not* the one you use to map in Dynaconf.

```python
>>> repository._worker_settings.get('AWS_ENDPOINT_URL')
'http://localstack:4566'
>>> repository._worker_settings.get('RSTUF_AWS_ENDPOINT_URL')
>>> 
``` 

We use `RSTUF_*` in docs for the user (Docker_README) but not in the code itself.